src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/DefaultRazorProjectHost.cs

```diff
@@ -268,16 +268,12 @@ internal async Task OnProjectChanged(IProjectVersionedValue<IProjectSubscription
             Item configurationItem, 
             out string[] configuredExtensionNames)
         {
-            if (!configurationItem.Value.TryGetValue(Rules.RazorConfiguration.ExtensionsProperty, out var extensionNames))
-            {
-                configuredExtensionNames = null;
-                return false;
-            }
-
+            // The list of extension names might not be present, because the configuration may not have any.
+            configurationItem.Value.TryGetValue(Rules.RazorConfiguration.ExtensionsProperty, out var extensionNames);
             if (string.IsNullOrEmpty(extensionNames))
             {
-                configuredExtensionNames = null;
-                return false;
+                configuredExtensionNames = Array.Empty<string>();
+                return true;
```

Comment:
Hmm actually, might want to handle this on the calling side. Technically this method is no longer a TryGet because we always return `true`. Should either change this to `GetExtensionNames` or handle the innability to find the extension names in the caller.